core/src/com/unciv/models/ruleset/tile/Terrain.kt

```diff
@@ -35,10 +35,17 @@ class Terrain : NamedStats() {
     var movementCost = 1
     var defenceBonus:Float = 0f
     var impassable = false
-    
-    @Deprecated("As of 3.14.1")
+
+    /**
+     * Do not use this directly! Use isRough() instead.
+     * Modded tiles may use the deprecated "Rough terrain" unique instead of this value, which is handled by isRough().
+     */
     var rough = false
```

Comment:
It seems to me that your implementation removes the uniques "Rough Terrain" and "Open Terrain", so this variable would be the only way to tell them apart. But it isn't set anywhere? How would that work?

Reply:
It is automagically set by the json parser.